blake3/Cargo.toml

```diff
@@ -6,4 +6,8 @@ license = "MIT OR Apache-2.0"
 
 [dependencies]
 p3-symmetric = { path = "../symmetric" }
-blake3 = "1.4.1"
+blake3 = "1.5"
+
+[features]
+neon = ["blake3/neon"]
+parallel = ["blake3/rayon"]
```

Comment:
Hm probably shouldn't be used with our Merkle trees since we parallelize over leaves, are you thinking of possible uses outside of Merkle trees?

Reply:
Yes, I agree. This was basically to expose the API just for it to be available. The `neon` feature is because of this remark in the Blake3 .toml:
https://github.com/BLAKE3-team/BLAKE3/blob/master/Cargo.toml#L15
which suggests they don't necessarily detect it, although it seems ok for `AArch64`.